experimental/sql/test_mlir/lit.cfg

```diff
@@ -0,0 +1,27 @@
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import lit.formats
+import os
+
+config.name = "Relation"
+config.test_format = lit.formats.ShTest(execute_external=True)
+config.suffixes = ['.ibis', '.xdsl']
+
+config.test_source_root = os.path.dirname(__file__)
+
+if "PYTHONPATH" in os.environ.keys():
+  config.environment[
+      "PYTHONPATH"] = config.test_source_root + "/../:" + os.environ[
+          "PYTHONPATH"]
+else:
+  config.environment["PYTHONPATH"] = config.test_source_root + "/../"
+
+config.environment[
+    "PATH"] = config.test_source_root + "/../tools/:" + os.environ["PATH"]
+
+config.available_features = []
+
+if sys.version_info[0] == 3 and sys.version_info[1] == 8:
+  config.available_features.append('python38')
```

Comment:
What does this do? Is this used in this PR? What about other Python versions?

Reply:
This is a fragment of a functionality we needed in a different repo. Basically, this checks that the python version is 3.8 (`sys.version_info[0] == 3 and sys.version_info[1] == 8`) and if so adds the `python38` feature. Then, in the tests one can use `REQUIRES python38` and the test will not be executed if the feature is not set.

I removed it, since it is not needed for this project.